cypress/integration/common/support/pageNameToPath.js

```diff
@@ -22,5 +21,8 @@ export const pageNameToPath = {
   'we cant help with your enquiry': '/not-eligible',
   'we find families in two ways': '/find-two-ways',
   'what help are you looking for today': '/what-help',
-  'where should we start looking': '/where-start-looking'
+  'eligible for one not sure': '/eligible-one-not-sure',
+  'eligible for one no active countries': '/eligible-one-no-active',
+  'where should we start looking': '/where-start-looking',
+  'eligible for one non public': '/eligible-one-not-public'
```

Comment:
I don't want to sound like a fusspot, but can I draw your attention to line 1 of this file ;-)

Reply:
done